crates/router/src/lib.rs

```diff
@@ -77,6 +77,7 @@ pub mod headers {
     pub const X_CLIENT_SOURCE: &str = "X-Client-Source";
     pub const X_PAYMENT_CONFIRM_SOURCE: &str = "X-Payment-Confirm-Source";
     pub const CONTENT_LENGTH: &str = "Content-Length";
+    pub const BROWSER_NAME: &str = "browsername";
```

Comment:
can you ensure that this key is case insensitive?

Reply:
Yes, I have verified it.